scripts/build-process/generate-pages-from-markdown.js

```diff
@@ -151,9 +151,9 @@ enrichedMarkdownFiles.forEach(({ category, fileName, title, subtitle, id }) => {
   });
 });
 
-const importsString = Object.keys(components).map(category =>
-  components[category].map(route => route.import).join('\n'),
-);
+const importsString = Object.keys(components)
+  .map(category => components[category].map(route => route.import).join('\n'))
+  .join('\n');
```

Comment:
This worked when we only had one category as the result was `[['import...', 'import...', 'import...']]`, but adding a category made it become ``[['import...', 'import...', 'import...']], ['import...', 'import...', 'import...']]`.

This change makes it still work as expected with multiple sub-arrays